package.json

```diff
@@ -1,5 +1,30 @@
 {
-  "name": "decidim-calafell",
+  "name": "app",
```

Comment:
It's not important, but maybe it's better to leave the old name.

Reply:
Too much copy pasta!